protocol/avdl/keybase1/kvstore.avdl

```diff
@@ -7,7 +7,7 @@ protocol kvstore {
     string teamName;
     string namespace;
     string entryKey;
-    string entryValue;
+    @nullSerializable(true) union { null, string } entryValue;
```

Comment:
this is the change. i created this decorator in https://github.com/keybase/node-avdl-compiler/pull/52 and used it here and then everything else is downstream.